tilequeue/command.py

```diff
@@ -1070,6 +1070,50 @@ def tilequeue_dump_tiles_of_interest(cfg, peripherals):
     )
 
 
+def tilequeue_load_tiles_of_interest(cfg, peripherals):
+    """
+    Given a newline-delimited file containing tile coordinates in
+    `zoom/column/row` format, load those tiles into a new Redis SET
+    alongside the existing one, then RENAME the new to the old.
+    """
+    logger = make_logger(cfg, 'load_tiles_of_interest')
+    logger.info('Loading tiles of interest')
+
+    toi_filename = "/Users/iandees/Downloads/toi.txt"
```

Comment:
I notice that the code to dump TOI uses an output filename of `"toi.txt"` in the current working directory. While I would also like to be able to pass the filename on the command line, would a pragmatic solution for now be to use the same name?